src/robot/model/testsuite.py

```diff
@@ -148,7 +148,7 @@ def setup(self) -> Keyword:
         return self._setup
 
     @setup.setter
-    def setup(self, setup: 'Keyword | None'):
+    def setup(self, setup: 'Keyword|None'):
```

Comment:
Good catch! As I mentioned above, we should add `Mapping` here as well. I let you to do that to avoid creating a conflict.